ShareX.UploadersLib/Forms/UploadersConfigForm.resx

```diff
@@ -11775,7 +11925,7 @@ Using an encrypted library disables sharing.</value>
     <value>4, 4, 4, 4</value>
   </data>
   <data name="ucLocalhostAccounts.Size" type="System.Drawing.Size, System.Drawing">
-    <value>792, 999</value>
+    <value>792, 2037</value>
```

Comment:
lol?

Reply:
What's wrong with that?

Reply:
Why does the control randomly doubles in height